### PR TITLE
Add roberta pooling layer to the trainable arguments

### PR DESCRIPTION
**Pull request by vid-koci**
Roberta randomly initialises the top pooling layer when loaded from transformers. Ideally, this layer should be trained in full, and not just with LoRa, to avoid random noise.

### CODE REVIEW

File: benchmark/data_frame_text_benchmark.py
Comment (zechengz) [inline benchmark/data_frame_text_benchmark.py:173]: Actually we don't use the pooling results not, instead we average over the last hidden states. Actually we can try to use the pooling layer results and also finetune the pooler.

**Review by zechengz — APPROVED** [review]
Left some comments, in general LGTM.

**Comment by vid-koci** [comment]
Let me, in that case, not merge this PR and we can add this if we actually decide to use the pooler. To avoid clutter in the code.

**Pull request closed without merging**
